src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -190,7 +197,35 @@ private function publishUpdate($object, array $targets): void
             $data = $this->serializer->serialize($object, key($this->formats), $context);
         }
 
-        $update = new Update($iri, $data, $targets);
-        $this->messageBus ? $this->dispatch($update) : ($this->publisher)($update);
+        $updates = array_merge([new Update($iri, $data, $targets)], $this->getGraphQlSubscriptionUpdates($object, $targets, $type));
+
+        foreach ($updates as $update) {
+            $this->messageBus ? $this->dispatch($update) : ($this->publisher)($update);
+        }
+    }
+
+    /**
+     * @param object $object
+     *
+     * @return Update[]
+     */
+    private function getGraphQlSubscriptionUpdates($object, array $targets, string $type): array
```

Comment:
It would mean introducing methods to get the created/updated/deleted items and to duplicate the code in `postFlush` and a little from `publishUpdate`. Is it worth it?